### ISSUE TITLE
Can't update author bio and job after deployment

### ISSUE THREAD

**Issue by Mmmu** [LouisBarranqueiro/tranquilpeak-hexo-theme#144]
@LouisBarranqueiro 
First of all, thanks for your great job! I like this theme very much, but I met a problem:

##### I've fill 'bio' and 'job' in language file, 'en.yml'.

in en.yml:
![11](https://cloud.githubusercontent.com/assets/13308521/11182612/00d9b76c-8ca7-11e5-9e24-1ebd2466eb2c.png)

##### When I browse that in localhost, the bio and job is fine. 
in my localhost:
![222](https://cloud.githubusercontent.com/assets/13308521/11182621/0adfa820-8ca7-11e5-8495-62f75d199b4b.png)

##### But after the deployment, It doesn't show correct info.
in my github page:
![image](https://cloud.githubusercontent.com/assets/13308521/11182481/2211178c-8ca6-11e5-9046-09b84d861e71.png)

You can visit it: mmmu.github.io

##### Here's the version info:
- Tranquilpeak:  1.4.3
- hexo:                3.1.1
- os:                    Windows_NT 10.0.10240 win32 x64
- http_parser:      2.5.0
- node:                4.2.2
- v8:                    4.5.103.35
- uv:                    1.7.5
- zlib:                   1.2.8
- ares:                 1.10.1-DEV
- icu:                   56.1
- modules:          46
- openssl:           1.0.2d

I'm using the user version, not the developer one.

**Comment by LouisBarranqueiro** [comment]
did you `run hexo generate --deploy` to deploy?

**Comment by Mmmu** [comment]
I did run:
- hexo clean
- hexo g
- hexo d

**Comment by Mmmu** [comment]
The 'bio' and 'job' shows right in my 'Mmmu.github.io' now.
I just sync my repo in github desktop for the changes of 'delete .gitattributes and .gitignore'

Before I use hexo,
I deleted these 2 files in my computer but didn't sync in github desktop, and then, I deleted these 2 files again on github.com manually. 
After these, I never open github desktop until several minutes ago.

I don't know why this works. 

Should I remove my page repo of github desktop in my computer ?
I feel there is some problem of sync with github desktop and hexo.

But anyway, things are good now. No more problem.
Thanks!!! :laughing:
